src/FileConventions/Library.fs

```diff
@@ -411,5 +411,31 @@ let ProjFilesNamingConvention(fileInfo: FileInfo) =
     fileName <> parentDirectoryName
 
 let NotFollowingNamespaceConvention(fileInfo: FileInfo) =
-    printfn "%s" fileInfo.FullName
-    false
+    assert (fileInfo.FullName.EndsWith(".fs"))
+
+    let fileName = Path.GetFileNameWithoutExtension fileInfo.FullName
+
+    let parentDirectoryName =
+        Path.GetDirectoryName fileInfo.FullName |> Path.GetFileName
+
+    printfn
+        "File name: %s, Parent directory name: %s"
+        fileName
+        parentDirectoryName
+
+    if parentDirectoryName <> "src"
```

Comment:
I know the logic is based on the corresponding issue, but for instance in NX we have multiple projects in "tests" folder that this rule can be applied to them. Can't we modify the rule to search for the folders containing projects?

Reply:
later